.openpublishing.redirection.json

```diff
@@ -0,0 +1,9 @@
+{
+  "redirections": [
+    {
+      "source_path": "Odata-docs/odatalib//v7/using-ut8jsonwriter-for-better-performance.md",
+      "redirect_url": "/odata/odatalib/using-utf8jsonwriter-for-better-performance",
```

Comment:
Does the article in question place the emphasis on `Utf8JsonWriter` or on `ODataUtf8JsonWriter`? If it's the latter, I'd rather you standardize the naming around `odatautf8jsonwriter` instead of `utf8jsonwriter`. `Utf8JsonWriter` is an implementation detail. We could argue `ODataUtf8JsonWriter` is an implementation detail too since its internal but I'd rather we pitch the type defined in our own library rather than in an external library. There's no problem though in mentioning that it rides on the `Utf8JsonWriter`.

Note also that based on the TOC, the new file you introduced is named _using-odatautf8jsonwriter-to-improve-serialization-performance_

Reply:
Technically `ODataUtf8JsonWriter` is also an implementation detail since it's not public. But customers are likely to be aware of `Utf8JsonWriter` since it's part of .NET's standard lib. But I agree it's an implementation detail. Not sure which one is emphasize in public docs.